src/components/Table/Internal/OcTable.tsx

```diff
@@ -88,6 +88,7 @@ function OcTable<RecordType extends DefaultRecordType>(
   props: OcTableProps<RecordType>
 ) {
   const {
+    bordered,
```

Comment:
for sake of typing, should you default to false rather than keeping undefined?

Reply:
sure

Reply:
Will do this in the HOC